### ISSUE TITLE
Detaches from current session when canceling selection via ctrl+c

### ISSUE THREAD

**Issue by senft** [LukeWood/tz#1]
hey, nice little util. i noticed that it detaches from the current session when you cancel the selection via ctrl+c.

**Comment by LukeWood** [comment]
Yeah that’s true, we could fix this by passing the current session name as a CLI argument to the detach script and using that as a default to reattach.  Or by swapping the order we do the detachment and the selection.
